src/http/input/metadata/LinkRelParser.ts

```diff
@@ -25,23 +25,13 @@ export class LinkRelParser extends MetadataParser {
   public async handle(input: { request: HttpRequest; metadata: RepresentationMetadata }): Promise<void> {
     const link = input.request.headers.link ?? [];
     const entries: string[] = Array.isArray(link) ? link : [ link ];
```

Comment:
Due to the changes in `parseLinkHeader` these can then be removed (both here and in the linkrel parser). Which does indeed make both this class quite small so perhaps just put the for loop in the handle directly 😄

Reply:
Ah yes, forgot about that of course..